src/scss/_Card.scss

```diff
@@ -15,3 +15,21 @@
 		transform: scale(1.05);
 	}
 }
+
+.Card div {
+	display: flex;
+	justify-content: center;
+	align-items: center;
+}
+
+.favorite {
+	margin-right: 10px;
+	width: 25px;
+	height: 25px;
+	align-items: center;
+
+	&:hover {
+		width: 30px;
+		height: 30px;
+	}
```

Comment:
There is a css method called transform? that allows you to do this same thing based on a percentage. I really like how you did it tho and think it is more readable.